### PR TITLE
feat: Added GitHub action to auto assign mentor for reviewal

### LINKED ISSUE #8: Add a Github Action to auto-assign Mentors for every PR
## Description 

Add a Github Action to auto-assign Mentors for every PR. It is a pain sometime to take a look at all the Projects for the PRs made by the Contributors. Getting a nice notification about reviewing a PR would make things easy for me. 

You can take a look at this [Github Action](https://github.com/marketplace/actions/auto-assign-action) for the purpose. Check [this](https://medium.com/better-programming/github-templates-the-smarter-way-to-formalize-pull-requests-among-development-teams-89f8d6a204f) out to know how to achieve the same.

### PR DESCRIPTION
**Pull request by m-code12**
Fixes #8

### CODE REVIEW

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+# Set to true to add assignees to pull requests
+addAssignees: false
+# A list of reviewers to be added to pull requests (GitHub user name)
+reviewers:
+  - HarshCasper
```
Comment (HarshCasper) [inline .github/auto_assign.yml]: Add a new line after this. Github sometimes raises an Issue due to this

File: .github/workflows/action.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: 'Auto Assign'
+on: pull_request
+jobs:
+  add-reviews:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kentaro-m/auto-assign-action@v1.1.2
```
Comment (HarshCasper) [inline .github/workflows/action.yml]: Add a new line after this. Github sometimes raises an Issue due to this

**Review by HarshCasper — CHANGES_REQUESTED** [review]
You can make these changes and squash the Commits before pushing. The implementation looks good to me 🥳

**Comment by m-code12** [comment]
I have made all the changes and squashed my commits @HarshCasper

**Comment by HarshCasper** [comment]
Thanks for your contribution @m-code12 🎆

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**